tests/api/runtimes/test_kubejob.py

```diff
@@ -178,3 +178,12 @@ def my_func(context):
 
         self._execute_run(runtime)
         self._assert_pod_create_called(expected_code=expected_code)
+
+    def test_set_env(self, db: Session, client: TestClient):
+        runtime = self._generate_runtime()
+        env = {"MLRUN_LOG_LEVEL": "DEBUG", "IMAGE_HEIGHT": "128"}
+        runtime.set_env("MLRUN_LOG_LEVEL", "DEBUG")
+        runtime.set_env("IMAGE_HEIGHT", "128")
+
```

Comment:
Better to define `env` as you did and then reference it. This way if you want to change something in its values you don't need to do it in two places. Replace the `set_env` lines with something like:
```
for env_variable in env:
    runtime.set_env(env_variable, env[env_variable])
```